templates/zerver/help/quote-and-reply.md

```diff
@@ -16,16 +26,15 @@ start a new topic.
 
 {end_tabs}
 
-You can also use the keyboard shortcut `>` to initiate quote-and-reply.
+!!! keyboard_tip ""
+
+    You can also use <kbd>Shift</kbd> + <kbd>.</kbd> to initiate
+    **Quote and reply**.
```

Comment:
Maybe "to initiate **Quote and reply** to the selected message." ? I feel like it's not quite obvious what you're replying to.

Reply:
After looking at this, I went with:

> You can also use `>` to **quote and reply** to the selected message.

I realized it didn't make sense to capitalize 'quote' in this context, but I left the bold formatting to connect to the article title and concept.